data/starts.txt

```diff
@@ -59,6 +70,128 @@ conversation "default intro"
+			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
+		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
+	scene "scene/reception"
+	`"I know you've wanted to be the captain of a starship for a long time now," your father says. "So as my birthday gift to you, how about we go get you a pilot's license and a ship from the shipyard?" You say yes without a second thought.`
+	`	Your father brings you to the Republic Department of Starship Licensing building in the spaceport. The lines are long, but your father walks right up to one of the desks. Before the person in the front of the line or the one behind the desk can complain, he tosses a credit chip worth a substantial amount at each of them. "I'd like to get a pilot's license for my child."`
+	`	"I'm sorry," the person behind the desk starts saying, "but you'll need to go through the process of applying through the net first, then wait three to five business days before-"`
+	`	Your father tosses another credit chip at the person behind the desk, prompting them to hand you a stylus and a pad. "Sign here, please. Your pilot's license will be here in a moment."`
+	`	Without any hesitation and after skipping past a number of pages of legal text, you sign your name:`
+	name
+	`	You enter the shipyard, your brand new pilot's license held tight in one hand and 490,000 credits from your father in the other. There are almost a dozen ship models for sale, but there are only three that you are able to afford. Which one you choose will determine your future...`
+
+
+
+start "syndicate"
+	name "Escaping Squalor"
+	description `You grew up on Earth, an overpopulated world with few opportunities. But you took a contract working for the Syndicate, and they gave you a place to live, enough to eat, and a decent wage off of Earth. Afterward you moved to Antipode, a cheap frontier world, but found that it wasn't any better than Earth. Fed up, you used what you had left to travel to the closest planet with a shipyard...`
+	description `	This is a more difficult start, with larger daily repayments.`
+	thumbnail "scene/earth"
+	date 16 11 3013
+	system "Scheat"
+	planet "Delve"
+	conversation "syndicate intro"
+	account
+		credits 480000
+		score 200
+		mortgage Mortgage
+			principal 530000
```

Comment:
Either this should be changed to 630000 or the below description should say 530,000.